amundsen_application/proxy/issue_tracker_clients/jira_client.py

```diff
@@ -148,5 +141,22 @@ def _generate_remaining_issues_url(self, table_uri: str, issues: List[DataIssue]
             return ''
         # jira expects a ticket key in the query to default to, so pick the first one
         first_issue_key = issues[0].issue_key
-        search_query = urllib.parse.quote(SEARCH_STUB.format(table_key=table_uri))
+        search_query = urllib.parse.quote(SEARCH_STUB_ALL_ISSUES.format(table_key=table_uri))
         return f'{self.jira_url}/browse/{first_issue_key}?jql={search_query}'
```

Comment:
Is this url correct? I tried locally and this didn't take me to a link where I could view all issues, instead it took me to the most recent issue.

Reply:
Updated link works for me